stories/Atom/Buttons-and-labels/Cta_button/buttons.stories.mdx

```diff
@@ -0,0 +1,200 @@
+import { Primarybuttonarrow } from './primary_button_arrow';
+import { Primarybuttonnoarrow } from './primary_button_noarrow';
+import { Primarybtnarrowdisbled } from './primary_button_arrow_disbled';
+import { Primarybtnnoarrowdisbled } from './primary_button_noarrow_disbled';
+import { Secondarybutton } from './secondary_button';
+import { Secondarybuttondisbled } from './secondary_button_disabled';
+import { Ctaarrow } from './cta_link_arrow';
+import  { Ctaspace } from './cta_link_space';
+
+export const getCaptionForLocale = (locale) => {
+  switch(locale) {
+    case 'english': const engText = {detail1: 'READ MORE', detail2: 'VIEW MORE'}; return engText;
+    case 'arabic': const arabicText = {detail1: 'اقرأ أكثر', detail2: 'عرض المزيد'}; return arabicText;
+    case 'burmese': const burmeseText = {detail1: 'ပိုပြီးဖတ်ပါ', detail2: 'နောက်ထပ်ကြည့်ပါ'}; return burmeseText;
+    case 'japanese': const japaneseText = {detail1: '続きを読む', detail2: 'vもっと見る'}; return japaneseText;
+    default:
+      return {detail1: 'READ MORE', detail2: 'VIEW MORE'};
+  }
+};
+
+<Meta title="Atoms/Buttons" component={Primarybuttonarrow} />
+
+<style>{`
+  div#story--atoms-buttons--primary-arrow, div#story--atoms-buttons--primary-arrow-disabled, div#story--atoms-buttons--primary-no-arrow, div#story--atoms-buttons--primary-no-arrow-disabled, div#story--atoms-buttons--secondary, div#story--atoms-buttons--secondary-disabled, div#story--atoms-buttons--cta-link-arrow, div#story--atoms-buttons--cta-link-space {
+    display: none;
+  }
+  .cta_wrapper {
+    display: flex;
+  }
+  .cta__label {
+    padding-right: 20px;
+  }
+  .cta__code {
+    padding-right: 16px;
+  }
+  .cta_item {
+    padding: 10px;
+  }
+  .subheading {
+    font-weight: 700;
+    font-size: 18px;
+    color: #666;
+    letter-spacing: 1.5px;
+    line-height: 24px;
+    margin-bottom: 12px;
+    margin-top: 40px;
+  }
+`}</style>
+
+# Buttons and CTA Component
+Cta buttons express what action will occur when the user clicks or touches it. Buttons are used to initialize an action, either in the background or foreground of an experience.
+<ul>
+  <li>Buttons are clickable elements that are used to trigger actions. They communicate calls to action to the user and allow users to interact with pages in a variety of ways. Button labels express what action will occur when the user interacts with it.</li>
+  <li>CTA-Button are divided into two options - Primary and Secondary cta.</li>
+</ul>
+
+<div className="subheading">When to use </div>
+Buttons are used primarily on action items. Some examples include Add, Read more, Save, Delete, View more and Sign up. Each page can have one or two primary or accent buttons. Any remaining calls-to-action should be represented as secondary buttons.
+
+<div className="subheading">When not to use </div>
+Do not use buttons as navigational elements. Instead, use links when the desired action is to take the user to a new page.
+
```

Comment:
@pallavikadam-srijan Please update the documentation by referring the below points-
The .btn classes are designed to be used with the <a>

Mention about other btn-classes, we are using

When using button classes on <a> elements that are used to trigger in-page functionality (like collapsing content), rather than linking to new pages or sections within the current page, these links should be given a role="button" to appropriately convey their purpose to assistive technologies such as screen readers.